.gitignore

```diff
@@ -8,6 +8,8 @@
 
 ## FILES
 .DS_Store
+# Custom environment for docker-compose (used by docker-compose.yml)
+/.env
```

Comment:
Niiice! I remember there was something funky about this file not being picked up automatically because we use `-f docker/docker-compose.yml`. Seems like they've fixed it.

Reply:
Seems to work for me now… hopefully the funkiness is gone :)

Reply:
If root `.env` is respected now, we can probably get rid of `./docker/.env` + `./docker/.default` -setup, but that's for another PR.